### PR TITLE
Streamline multi repo checkout

### PR DESCRIPTION
**Pull request by RDPerera**
## Purpose

Fixes:

## Checklist

- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,10 +17,13 @@ jobs:
...
     steps:
       - name: Checkout Repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+        with:
+          path: jco
 
       - name: Get Ballerina Version
         run: |
+          cd jco
```
Comment (niveathika) [inline .github/workflows/ci.yml:26]: Shall we use working-dir config for an action

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -44,22 +47,22 @@ jobs:
           echo -e '${{ toJson(secrets) }}' | jq -r 'to_entries[] | .key + "=" + .value' >> $GITHUB_ENV
 
       - name: Clone lib repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           repository: wso2-enterprise/wso2-sap-resources
           token: ${{ secrets.BALLERINA_BOT_TOKEN }}
-          path: 'wso2-sap-resources'
+          path: 'libs'
```
Comment (niveathika) [inline .github/workflows/ci.yml:54]:
```suggestion
          path: libs
```

File: .github/workflows/daily-build.yml [hunk 3]
```diff
@@ -31,22 +33,22 @@ jobs:
           echo -e '${{ toJson(secrets) }}' | jq -r 'to_entries[] | .key + "=" + .value' >> $GITHUB_ENV
```
Comment (niveathika) [inline .github/workflows/daily-build.yml:33]: No need

File: .github/workflows/daily-build.yml [hunk 4]
```diff
@@ -31,22 +33,22 @@ jobs:
...
 
       - name: Build the Package
         env:
           packageUser: ${{ github.actor }}
           packagePAT: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          cd jco
           ./gradlew build -x test -PbuildUsingDocker=nightly
 
       # Send notification when build fails
```
Comment (niveathika) [inline .github/workflows/daily-build.yml:54]: Only notification is for daily build. Remove it from ci workflow

**Comment by niveathika** [comment]
Lets add the PR workflow too

File: .github/workflows/daily-build.yml (hunk 4 above)
Comment (RDPerera) [reply .github/workflows/daily-build.yml:50]: Fixed with [12e315e](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/8/commits/12e315eb1c841256838179c834406987a7fbff59).

File: .github/workflows/daily-build.yml (hunk 3 above)
Comment (RDPerera) [reply .github/workflows/daily-build.yml]: Changed with [12e315e](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/8/commits/12e315eb1c841256838179c834406987a7fbff59).

File: .github/workflows/ci.yml (hunk 2 above)
Comment (RDPerera) [reply .github/workflows/ci.yml]: Changed with [12e315e](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/8/commits/12e315eb1c841256838179c834406987a7fbff59).

**Comment by RDPerera** [comment]
> Lets add the PR workflow too

Added with https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/9.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (RDPerera) [reply .github/workflows/ci.yml]: Updated

**Review by niveathika — APPROVED** [review]

**Pull request merged**